### PR TITLE
Packages: moved systemd service to forking on rpm-based distros

### LINKED ISSUE #915: application logging to files not working for php wordpress
even though we setup as in doc: https://unit.nginx.org/news/2023/unit-1.30.0-released/#application-logging

still php error log goes to main unit.log file

```
"testsite": {
		"stderr": "/tmp/unit_app.log",
		"stdout": "/tmp/unit_app_error.log",
		"user": "unit",
		"type": "php 7.4",
		"targets": {
			"direct": {
				"root": "/sites/site/"
			},

			"index": {
				"root": "/sites/site/",
				"script": "index.php"
			}
		}
	}
```
thanks

```
# unitd --version
unit version: 1.30.0
configured as ./configure --prefix=/usr --statedir=/var/lib/unit --control=unix:/var/run/unit/control.sock --pid=/var/run/unit/unit.pid --log=/var/log/unit/unit.log --tmpdir=/var/tmp --user=unit --group=unit --tests --openssl --njs --modulesdir=/usr/lib64/unit/modules --libdir=/usr/lib64 --cc-opt='-O2 -flto=auto -ffat-lto-objects -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -march=x86-64-v2 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -fPIC' --ld-opt='-Wl,-z,relro -Wl,-z,now -pie'
```

### LINKED ISSUE #1178: Broken stdout and stderr logging in Amazon Linux
Systemd unit in package unit-1.32.0-1.amzn2023.ngx.x86_64.rpm contains this line: `/usr/sbin/unitd $UNITD_OPTIONS --no-daemon` which renders `stdout` and `stderr` options of `/config/applications/*` objects useless, according to this line in the documentation:
> When running in --no-daemon mode, application output is always redirected to Unit’s log file.

### PR DESCRIPTION
**Pull request by thresheek**
Fixes #915
Fixes #1178

### CODE REVIEW

**Review by ac000 — COMMENTED** [review]
Hi @thresheek 

Please replace the Fixes tags with

```
Closes: https://github.com/nginx/unit/issues/915
Closes: https://github.com/nginx/unit/issues/1178
```

We use `Fixes` for when referring to an actual commit, e.g

`Fixes: c1e3f02f9 ("Compile with -fno-strict-overflow")`

And use full URLs so they are actionable from outside GH, e.g from a terminal.

(Note: While we normally put URLs inside `<>` we don't for these tags as GH doesn't recognise them if we do)

**Comment by thresheek** [comment]
Thanks, I've fixed the commit message as per your guidelines.

File: pkg/rpm/rpmbuild/SOURCES/unit.service [hunk 1]
```diff
@@ -15,12 +15,13 @@ Wants=network-online.target
...
 [Service]
-Type=simple
+Type=forking
 Environment="UNITD_OPTIONS=--log /var/log/unit/unit.log --pid /var/run/unit/unit.pid"
-ExecStart=/usr/sbin/unitd $UNITD_OPTIONS --no-daemon
+ExecStart=/usr/sbin/unitd $UNITD_OPTIONS
 ExecReload=
 RuntimeDirectory=unit
 RuntimeDirectoryMode=0755
+PIDFile=/var/run/unit/unit.pid
```
Comment (ac000) [inline pkg/rpm/rpmbuild/SOURCES/unit.service:24]: Can technically just be `/run/unit/unit.pid`

`/var/run` has been a symlink to `/run` for I don't know how many years... but no biggie...

But yeah, looks good.

**Review by ac000 — APPROVED** [review]

File: pkg/rpm/rpmbuild/SOURCES/unit.service (hunk 1 above)
Comment (thresheek) [reply pkg/rpm/rpmbuild/SOURCES/unit.service]: Oh yeah, let's make it contemporary.  I've checked Amazon Linux 2 and CentOS 7, and they already have a symlink, so we should just fix that (which I just did).

**Pull request merged**